physics/rotating_pulsating_reference_frame_body.hpp

```diff
@@ -172,9 +189,15 @@ RotatingPulsatingReferenceFrame<InertialFrame, ThisFrame>::r_derivatives(
         ephemeris_->trajectory(secondary)->EvaluatePosition(t),
         secondary->gravitational_parameter());
   }
+  BarycentreCalculator<Position<InertialFrame>, GravitationalParameter>
+      primary_position;
```

Comment:
Would fit on the previous line?  (Many places in this PR.)

Reply:
No.